gitops/components/kubevirt/kustomization.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- https://github.com/kubevirt/kubevirt/releases/download/v1.1.0/kubevirt-cr.yaml
+- https://github.com/kubevirt/kubevirt/releases/download/v1.1.0/kubevirt-operator.yaml
+- https://github.com/kubevirt/common-instancetypes/VirtualMachineClusterInstancetypes
+- https://github.com/kubevirt/common-instancetypes/VirtualMachineClusterPreferences
+- ./resources.yaml
+
+patches:
+  - patch: |-
+      - op: add
+        path: /spec/configuration/developerConfiguration/useEmulation
+        value: true
+    target:
+      kind: kubevirt
```

Comment:
Is there actually a Kubernetes object of `kind: kubevirt`? My guess is that this is a typo.